File: src/test/java/ninetynineproblems/lists/P01Test.java
package ninetynineproblems.lists;

import static java.util.Arrays.asList;
import static org.hamcrest.CoreMatchers.equalTo;
import static org.hamcrest.CoreMatchers.is;
import static org.junit.Assert.assertThat;

import org.junit.Test;

public class P01Test {

  @Test
  public void shouldFindLastElementFromAListOfAlphabets() throws Exception {
    assertThat(P01.last(asList("a", "b", "c", "d")), is(equalTo("d")));
  }
}

File: src/test/kotlin/ninetynineproblems/lists/P01Test.kt
package ninetynineproblems.lists

import org.junit.Test
import kotlin.test.assertEquals

class P01TestKt {
  @Test
  fun shouldFindLastElementFromAListOfAlphabets() {
    assertEquals(last(listOf("a", "b", "c", "d")), "d")
  }
}
